modin/core/io/file_dispatcher.py

```diff
@@ -276,14 +274,21 @@ def file_exists(cls, file_path, storage_options=None):
                 else:
                     new_storage_options = {}
 
-                s3fs = S3FS.S3FileSystem(anon=False, **new_storage_options)
-                exists = False
+                fs, _ = fsspec.core.url_to_fs(file_path, **new_storage_options)
+                is_exist = False
```

Comment:
why `is_exist` vs `exists`?

Reply:
I prefer to start naming of boolean values by `is`/`can`/`has`/etc because it additionally highlights type of variable and looks more clearer in subsequent code.

Reply:
Could we try `does_exist`? I think that would flow better here :D

Reply:
Why would it be better? We already use approach when naming starts from `is_` for boolean values/methods/properties (functions from pandas like [these](https://github.com/modin-project/modin/blob/05bf659b67d2b7153d8aaa004dd95252377eed51/modin/core/storage_formats/pandas/query_compiler.py#L28-L31), for example)

Reply:
Grammar wise I think `does_exist` or even just `exists` is technically more correct than `is_exist`. We could do like `is_valid` or something like that if you want to stick with the `is_*`

Reply:
I'm not a native speaker so I reverted previous variant of naming for this variable (`exists`), if this is more grammatically correct

Reply:
No worries! Not a problem at all :D